NET.A.Carting/Application/Common/Interfaces/IMessageBusConsumer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Application.Common.Interfaces
+{
+    public interface IMessageBusConsumer
+    {
+        Task ReceiveMessage<T>(string queueName);
+
+        Task SetupMessageConsumersAsync();
+
+        void Dispose();
```

Comment:
Inherit interface from IDisposable instead of adding this method manually.

Reply:
The interface IMessageProducer is IDisposable